### ISSUE TITLE
3 keys event_mapping won't work

### ISSUE THREAD

**Issue by kurokirasama** [curlpipe/ox#132]
**Describe the bug**
when defining key binds with 3 keys, for instance `ctrl_shift_h`, the function for that key bind won't run, even more I think the key bind isn't recognized.

**To Reproduce**
For instance:
```lua
-- duplicate the current line
    ["ctrl_shift_d"] = function()
        local current_line = editor:get_line()
        editor:insert_line_at(current_line, editor.cursor.y + 1)
        editor:move_down()
    end,
```
when pressing `ctrl_shift_d` the `ctrl_d` activates.

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] ubuntu 24.04, ox 0.4.2

**Comment by curlpipe** [comment]
From what I remember, this issue was raised previously.

I think it's to do with the library `crossterm` not supporting it by default (see the issue I opened [here](https://github.com/crossterm-rs/crossterm/issues/515) )

It looks like there is a way to do it on certain terminals with [this](https://docs.rs/crossterm/latest/crossterm/event/struct.PushKeyboardEnhancementFlags.html).

I'll have a go at implementing this.

**Comment by curlpipe** [comment]
Fixed in 0.4.3
